packages/mobile/App.svelte

```diff
@@ -25,12 +26,36 @@
     import { initialiseRouterManager, RouterManagerExtensionName } from '@core/router'
 
     import { DashboardView, LoginRouter, OnboardingRouter } from '@views'
-    import { closeAllDrawers } from '@/auxiliary/drawer'
+    import { closeAllDrawers, drawers } from '@/auxiliary/drawer'
 
     appStage.set(AppStage[process.env.STAGE.toUpperCase()] ?? AppStage.ALPHA)
 
     checkAndMigrateProfiles()
 
+    /**
+     * Handle Android top status bar (not needed for iOS)
+     * @todo remove when implement status bar overlay
```

Comment:
does this mean that al this code will we removed when we implement issue `6345`?

Reply:
Exactly, when we'll be using overlay status bar mode we can use safe-areas on Android and forget about changing colors and theme on the status bar.